<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="SourceBrowser" name="sourceBrowser">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragOnly</enum>
       </property>
      </widget>
      <widget class="ViewsTabWidget" name="ViewsTabs">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="TextViewTab">
        <attribute name="title">
         <string>Text</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="ObjectTextViewer" name="textViewer"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="BinaryViewTab">
        <attribute name="title">
         <string>Binary</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QHexEdit" name="hexEditor" native="true">
             <zorder>sourceBrowser</zorder>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="GraphViewTab">
        <attribute name="title">
         <string>Graph</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="GraphView" name="graphViewer">
             <property name="interactive">
              <bool>true</bool>
             </property>
             <property name="dragMode">
              <enum>QGraphicsView::RubberBandDrag</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenInTextViewer"/>
    <addaction name="actionOpenInBinaryView"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenInTextViewer">
   <property name="text">
    <string>Open in Text Viewer</string>
   </property>
  </action>
  <action name="actionAddToScene">
   <property name="text">
    <string>Add to scene</string>
   </property>
  </action>
  <action name="actionOpenInBinaryView">
   <property name="text">
    <string>Open in Binary View</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphView</class>
   <extends>QGraphicsView</extends>
   <header>graphview.h</header>
  </customwidget>
  <customwidget>
   <class>QHexEdit</class>
   <extends>QWidget</extends>
   <header>widgets/qhexedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ObjectTextViewer</class>
   <extends>QTextEdit</extends>
   <header>widgets/ObjectTextViewer.h</header>
  </customwidget>
  <customwidget>
   <class>SourceBrowser</class>
   <extends>QTreeView</extends>
   <header>widgets/SourceBrowser.h</header>
  </customwidget>
  <customwidget>
   <class>ViewsTabWidget</class>
   <extends>QTabWidget</extends>
   <header>widgets/ViewsTabWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>loadTextFile()</slot>
 </slots>
</ui>
